pagopa/pn-frontend
# update pagopa/mui-italia version (pn-8759, pn-7135, pn-6687)

## Short description
Update pagopa/mui-italia version to fix some bugs

## List of changes proposed in this pull request
- Changed packages version
- Applied custom style to timeline components

## How to test
- Log with pa and check that current product is selected in the dropdown and that, in mobile version, timeline has right style
- Log with pf and check that, in mobile version, icons in the header aren't grey
